server/src/index.ts

```diff
@@ -0,0 +1,9 @@
+export * from './utils';
```

Comment:
What does the import look like if you do it like this?

Reply:
tried like this import { setupFromConfigFile } from 'roku-test-automation';  and seems to resolve properly

Reply:
i think we can even import * as rta from 'roku-test-automation'; like this i believe

Reply:
Please let me know your thoughts i can even export methods individually too.

Reply:
Had a feeling that was how it would do it. So you can get name conflicts. You would want to export it as an object. Let me know if yup need help with the syntax

Reply:
I thought about the conflict but than thought we can always store to a object and use rta. Syntax. Cool will update the export to use objects

Reply:
So you either import individual modules or import everything as rta and do it that way it’s how someone chooses to consume it

Reply:
I like the approach of using rta so that we have only one approach of import. Please let me know if you ok with it?